### PR TITLE
Renamed writeOften to enableWriteBuffer

### PR DESCRIPTION
**Pull request by frznlogic**

### CODE REVIEW

File: agent/unit-test/softwarecontaineragent_unittest.cpp [hunk 1]
```diff
@@ -33,14 +33,15 @@ class SoftwareContainerAgentTest: public ::testing::Test
...
+    SoftwareContainerWorkspace *workspace;
 
     void SetUp() override
     {
-        sca = new SoftwareContainerAgent(m_context
-                , m_preloadCount
-                , m_shutdownContainers
-                , m_shutdownTimeout);
+        sca = new SoftwareContainerAgent(
+                    m_context
```
Comment (tobsan) [inline agent/unit-test/softwarecontaineragent_unittest.cpp]: This identation doesn't look very nice.

File: agent/unit-test/softwarecontaineragent_unittest.cpp [hunk 2]
```diff
@@ -33,14 +33,15 @@ class SoftwareContainerAgentTest: public ::testing::Test
     int m_preloadCount = 1;
     bool m_shutdownContainers = true;
     int m_shutdownTimeout = 2;
-    SoftwareContainerWorkspace workspace;
+    SoftwareContainerWorkspace *workspace;
```
Comment (tobsan) [inline agent/unit-test/softwarecontaineragent_unittest.cpp]: Please use unique_ptr or similar here.

**Comment by tobsan** [comment]
Looks good apart from the comments I've already given.

File: agent/unit-test/softwarecontaineragent_unittest.cpp (hunk 1 above)
Comment (breakreturn) [reply agent/unit-test/softwarecontaineragent_unittest.cpp]: Please take a look a the style guidelines.

**Comment by tobsan** [comment]
Awesome.

**Pull request merged**